python/pyspark/sql/readwriter.py

```diff
@@ -1259,12 +1218,15 @@ def parquet(self, path, mode=None, partitionBy=None, compression=None):
                 exists.
         partitionBy : str or list, optional
             names of partitioning columns
-        compression : str, optional
-            compression codec to use when saving to file. This can be one of the
-            known case-insensitive shorten names (none, uncompressed, snappy, gzip,
-            lzo, brotli, lz4, and zstd). This will override
-            ``spark.sql.parquet.compression.codec``. If None is set, it uses the
-            value specified in ``spark.sql.parquet.compression.codec``.
+
+        Other Parameters
+        ----------------
+        Extra options
+            For the extra options, refer to
+            `Data Source Option <https://spark.apache.org/docs/latest/sql-data-sources-parquet.html#data-source-option>`_  # noqa
+            and
+            `Generic File Source Options <https://spark.apache.org/docs/latest/sql-data-sources-generic-options.html`>_  # noqa
```

Comment:
Here you linked generic option but you didn't at https://github.com/apache/spark/pull/32161/files#diff-e283917d906671f2e032d1debe091cb13038440e2daa24b982ce5d486d96b0d7R863-R866